docs/_articles/en/v3.x/getting-started/install.md

```diff
@@ -1,45 +1,47 @@
 ---
-title: Install Salesforce Code Analyzer Plugin
+title: Install Salesforce Code Analyzer
 lang: en
 ---
 
-## Install v3.x
+## Install Salesforce Code Analyzer v3.x
 
-Like all the Salesforce CLI plug-ins, you install the Salesforce Code Analyzer CLI plug-in with one simple step. NOTE: Be sure you've completed the [prerequisites](./en/v3.x/getting-started/prerequisites/) before you install this plug-in.
-
-
-By default, `latest` tag gets installed and as of today, it still points to {{ site.data.versions.scanner }}. You can install the pilot version by pointing to the `latest-pilot` tag. Also, note that automatic plugin upgrades don't work here.
+**NOTE**: Complete the [prerequisites](https://forcedotcom.github.io/sfdx-scanner/en/getting-started/prerequisites/) before you install this plug-in. Automatic plug-in upgrades don't work with the pilot version 3.x.
```

Comment:
This entire file will need to change significantly, since the next 3.x release is not a pilot.